src/initializers/bull/bull-helper.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import isNil = require('lodash/isNil');
+
+export function queuePath(queueName: string): string {
+  return path.join(__dirname, '..', queueName);
+}
+
+export async function list(config): Promise<string[]> {
+  let queueNames = [];
+  if (config.bull.queues) {
+    queueNames = config.bull.queues.map((q) => q.name);
+  }
+  return queueNames;
+}
+
+export function encode(val): any {
+  isNil(val) ? val : Buffer.from(JSON.stringify(val)).toString('base64');
+}
```

Comment:
Nil and undefined values are falsy in javascript, so I don't think you need the `lodash.isNil`.

You could use javascripts's native `atob()` and `btoa()` methods. Also did you forget to return in this method? And keep in mind that that stringifying a string adds extra quotes, is that what you want?

May I suggest

```suggestion
export const encode = (val: any) =>  val && btoa(`${val}`);
```

Reply:
Thank you for the solution, I removed the method completely since I don't use it after all.